lib/constants/signuplogin_constants.dart

```diff
@@ -0,0 +1,25 @@
+//Welcome in the build
+const String welcomeMessage = "Welcome to Aspire";
+
+//Account Verify
+const String vertifyaccountTitle = "Verify your account";
+const String vertifyaccountMessage =
+    "Link to verify account has been sent to your email";
+const String dissmissMessage = "Dismiss";
+
+//Hint Text
+const String emailHint = "Email";
+const String passwordHint = "Password";
+
+//Primary Button
+const String loginButton = "Login";
+const String createAccountButtonPrimary = "Create Account";
+
+//Secondary Button
+const String createAccountButtonSecondary =
+    "Do not have account? Create account";
+const String haveAccountButton = "Have an account? Sign in";
+
+//Error Message
+const String emailemptyMessage = "Email can't be empty";
+const String passwordemptyMessage = "Password can't be empty";
```

Comment:
camelCase format for both of these string names please!

Reply:
Got it